db/seeds.rb

```diff
@@ -46,26 +46,26 @@ def build_orders
   end
 
   def build_items
-    item1 = Item.create(title: "Gut Buster", description: "If its in the store, it's in this burrito!", price: 1000, image: open("app/assets/images/burrito_360.png"))
-    item2 = Item.create(title: "Atomic Sunrise", description: "Spicy green chili, egg, and potatoe burrito.", price: 850, image: open("app/assets/images/burrito_360.png"))
-    item3 = Item.create(title: "Fiesta Especial", description: "A 6 foot party sized burrito to share with your family and friends.", price: 10000, image:open("app/assets/images/burrito_360.png"))
+    item1 = Item.create(title: "Gut Buster", description: "If it's in the store, it's in this burrito!", price: 1000, image: open("app/assets/images/burrito_360.png"))
+    item2 = Item.create(title: "Atomic Sunrise", description: "Spicy green chili, egg, and potato burrito.", price: 850, image: open("app/assets/images/burrito_360.png"))
+    item3 = Item.create(title: "Fiesta Especial", description: "A 6-foot party-sized burrito to share with your family and friends.", price: 10000, image:open("app/assets/images/burrito_360.png"))
```

Comment:
Useless assignment to variable - `item3`.<br>Line is too long. [196/80]<br>Space missing after colon.